### ISSUE TITLE
Qualification workflow: add local sensitivity analysis

### ISSUE THREAD

**Issue by sfrechen** [Open-Systems-Pharmacology/OSPSuite.ReportingEngine#790]
New feature request: local sensitivity analysis for (selected) parameters on selected PK parameters.

Output should look like this:
![image](https://user-images.githubusercontent.com/26569320/164652487-8061038e-6b01-4a7b-8355-bbab75645293.png)

or like this:
![image](https://user-images.githubusercontent.com/26569320/164652566-218b46ae-51fa-453c-8a39-5a6b68a81954.png)

The upper one is close to what we have in mean model workflows. The latter is only possible for selected model parameters.

**Comment by Yuri05** [comment]
I like the feature :)
Some remarks and questions:
1. Do we need those sensitivity plots only for **individual** simulations or also for **population** simulations? In case of population simulations, we could define the quantiles (like in the PopulationWorkflow), for which the sensitivity should be plotted
2. The sensitivity plot would be the same as in MeanModel-Workflow (resp. in Population-Workflow)
3. Configuration of sensitivity settings would be the same as in Sensitivity task of MeanModel/Population workflows
  * [OPTIONAL] List of sensitivity parameters. If not given: use default settings of the OSPSuite.Core (calculate sensitivity of all parameters which are non-formula, non-zero, non-categorical etc.)
  * [OPTIONAL] Threshold for cumulated sensitivity. Only parameters with the most contribution to the cumulated sensitivity below the threshold will be plotted. If not given: 0.9 if no sensitivity parameter list is specified, 1 if specified.
  * [OPTIONAL] Max. number of sensitivity parameters per plot
  * [OPTIONAL] Variation range and number of steps
  * [OPTIONAL] Quantiles for population simulations (default 5% - 50% - 95%)
4. The second plot seems to be exactly the same what was requested by @PavelBal for the ospsuite-R package (@PavelBal I could not find the feature request - where is it?)
5. Should we allow **wildcards** in the (optionally) passed sensitivity parameter names?

**Comment by msevestre** [comment]
>Should we allow wildcards in the (optionally) passed sensitivity parameter names?

This is the case in ospsuite-r already

**Comment by Yuri05** [comment]
>This is the case in ospsuite-r already 

But not in the Reporting Engine (@abdullahhamadeh @pchelle correct me if I am wrong) https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/issues/314

**Comment by Yuri05** [comment]
check if not (partly) implemented in esqLabsR
@PavelBal

**Comment by PavelBal** [comment]
The second type of plot ("spider plot") is indeed implemented - https://esqlabs.github.io/esqlabsR/articles/esqlabsR-sensitivity.html

The first one would be easy to add as the results are already available.

**Comment by PavelBal** [comment]
Available in ´{esqlabsR}`: https://esqlabs.github.io/esqlabsR/articles/sensitivity.html#sensitivity-tornado-plot

**Comment by PavelBal** [comment]
@Yuri05 close then?

**Comment by Yuri05** [comment]
> [@Yuri05](https://github.com/Yuri05) close then?

No. The feature is not yet available in the Qualification Framework.
